processed 2 tasks

task 1 'publish'. lines 5-27:
Error: Transaction Effects Status: Iota Move Bytecode Verification Error. Please run the Iota Move Verifier for more information.
Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: IotaMoveVerificationError, source: Some("_::M1. 'init' function must be private"), command: Some(0) } }
